x/cert/query/querier.go

```diff
@@ -0,0 +1,113 @@
+package query
```

Comment:
In this file there is lots of usage of `path[2]`,`path[3:]`, `path[3]` etc. It'd be easier to understand if those were named constants in the file.